.swiftlint.yml

```diff
@@ -40,7 +40,6 @@ disabled_rules:
   - no_magic_numbers
   - optional_enum_case_matching
   - prefer_nimble
-  - prefer_self_in_static_references
```

Comment:
There is no common sense among the collaborators of this project whether this rule is helpful. It was disabled intentionally. We should rather keep it inactive.

Reply:
Cool - I've re-disabled it.